### ISSUE TITLE
MarkerCluster: icon_create_function is not inserted as function

### ISSUE THREAD

**Issue by nakasix** [python-visualization/folium#945]
Hello,

I have a problem with my generation of map. In fact, I obtain double quotes in my source code, so I get "this._group.options.iconCreateFunction is not a function" message in Firefox.

Sample of my script :

```python
# Create Map
obj_map_cluster = folium.Map(
    # Latitude and Longitude of Map (Northing, Easting)
    location=[46.5286, 2.4389],
    # Forces Leaflet to use the Canvas back-end for vector layers instead of SVG
    prefer_canvas=True,
    # Map tileset to use
    tiles='cartodbdark_matter',
    # Initial zoom level for the map
    zoom_start=6
)

# Assign icon create function
obj_markercluster = plugins.MarkerCluster(
    icon_create_function="""
function (cluster) {
    return L.divIcon({ html: cluster.getChildCount(), className: 'mycluster', iconSize: L.point(40, 40) });
}
"""
)

# Create Marker
obj_marker = folium.Marker([DEFINITION OF MARKER])
obj_markercluster.add_child(obj_marker)
obj_markercluster.add_to(obj_map_cluster)
obj_map_cluster.save('sites_cluster.html')

```
#### Problem description

HTML source code is : 
`"iconCreateFunction": "function (cluster) {\n    return L.divIcon({ html: cluster.getChildCount(), iconSize: L.point(40, 40) });\n}"`

#### Expected Output

To work, I would like : 
`iconCreateFunction: function (cluster) {    return L.divIcon({ html: cluster.getChildCount(), iconSize: L.point(40, 40) });}`

Without '"' and '\n'

#### Output of ``folium.__version__``

My version is 0.6.0+11.g71ab988

Thank you !

**Comment by nakasix** [comment]
Hello,

To complete, I have no problem with version 0.5.0.

Regards,

**Comment by Conengmo** [comment]
Good catch, this is indeed a problem. It is because we added a possibility to pass options in 0.6. The iconCreateFunction is added to the options dictionary, converted to json, and then inserted into the template. For the function to work it must be inserted in the template directly.

I just worked with this in #916. There I added an intermediate function that merged the options object and the custom function, so there's one solution. 

Maybe another could be to extend the options object in place, something like this (with jQuery here):
`$.extend({{ this.options }}, {"iconCreateFunction": {{ this.icon_create_function }}})`

Can you help with this @nakasix?

**Comment by Conengmo** [comment]
I just used a much simpler way in #916, that could also work here.

**Comment by Conengmo** [comment]
This has been fixed in #954. To use it you should install folium from the git master branch (until we release a next version). Thanks @nakasix for reporting this. By the way, if you have time maybe you can confirm it works now.

**Comment by nakasix** [comment]
Hello,

Sorry for my absence, I confirm that it works with the master version.

Thank you,
